### ISSUE TITLE
Walsh documentation missing

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/reaver-wps#102]
```
0. What version of Reaver are you using?
svn rev 76

1. What operating system are you using (Linux is the only supported OS)?
linux Backtrack 5 RC1 VMWARE image official
Linux bt 2.6.39.4 #1 SMP Thu Aug 18 13:38:02 NZST 2011 i686 GNU/Linux

2. Is your wireless card in monitor mode (yes/no)?
yes  -> used airmon-ng start wlan0
3. What is the signal strength of the Access Point you are trying to crack?

Quality=61/70  Signal level=-49 dBm 

4. What is the manufacturer and model # of the device you are trying to
crack?

Netgear WNR3500v2
5. What is the entire command line string you are supplying to reaver?
NA
6. Please describe what you think the issue is.
Basically walsh works as expected. But it would be nice to have some defintions 
what is listed and a howto.
For example. I have no idea what is meant with "WPS Locked", does it mean the 
WPS is not accepting any pins or is it rate limiting? 
SSID                  Channel       WPS Version       WPS Locked        ESSID
--------------------------------------------------------------------------------
--------------
E0:91:F5:A6:33:37       3            1.0               N                 A


```

Original issue reported on code.google.com by `erick.va...@gmail.com` on 7 Jan 2012 at 2:27

**Comment by GoogleCodeExporter** [comment]
```
WPS locked means that the AP is reporting that it has locked WPS and is not 
accepting any pins. This could be the result of rate limiting, or it could be 
an option the administrator enabled/disabled on the AP. It could be that the AP 
is actually still accepting pins, but just reporting that it isn't. There is 
really no way for walsh to know what the reason is, so it just displays what 
the AP is reporting.

Documentation is in progress and will be included in the next release.
```

Original comment by `cheff...@tacnetsol.com` on 7 Jan 2012 at 2:54
- Changed state: **Started**
- Added labels: **Type-Enhancement**, **Priority-Low**
- Removed labels: **Type-Defect**, **Priority-Triage**

**Comment by GoogleCodeExporter** [comment]
```
thanks for the explanation.
```

Original comment by `erick.va...@gmail.com` on 7 Jan 2012 at 2:56

**Comment by GoogleCodeExporter** [comment]
```
Documentation added in r87.
```

Original comment by `cheff...@tacnetsol.com` on 11 Jan 2012 at 6:56
- Changed state: **Fixed**
